package.json

```diff
@@ -133,6 +135,16 @@
         "title": "Expand Alias",
         "category": "PowerShell"
       },
+      {
+        "command": "PowerShell.SetISECompatibility",
+        "title": "Set ISE like compatibility",
```

Comment:
I was thinking

"Enable ISE Mode"

What do you think of that?

Reply:
That sounds better than what I originally had. My original was "Make ISE-like" and "Make Default-like" so enable and disable is much better.